Octokit.Tests/Clients/AuthorizationsClientTests.cs

```diff
@@ -37,6 +37,19 @@ public void GetsAListOfAuthorizations()
                     Arg.Is<Uri>(u => u.ToString() == "authorizations"),
                     Args.ApiOptions);
             }
+
+            [Fact]
+            public void RequestsCorrectUrlWithApiOptions()
+            {
+                var client = Substitute.For<IApiConnection>();
+                var authEndpoint = new AuthorizationsClient(client);
+                
+                authEndpoint.GetAll(ApiOptions.None);
+
+                client.Received().GetAll<Authorization>(
+                    Arg.Is<Uri>(u => u.ToString() == "authorizations"),
+                    Args.ApiOptions);
```

Comment:
This test and the one before it (`RequestsCorrectUrl()`) are both checking the same thing.  Perhaps this one should check that the specific ApiOptions object specified was passed through to the client.  Eg instead of using ApiOptions.None, declare one containing some page size/skip settings, then verify it is actually seen on the `Received()` call?

I realise you were just following the reference implementation but Im just wondering before we go through and implement ALL the pagination PR's we should settle on a good set of test coverage.  There are only 1 or 2 merged so far, so we can easily go back and add the extra unit test to those if we decide it's needed.  The main thing is, it would be done on all the remaining issues as they are merged.

My 2c is that as much as possible gets covered by the unit tests! :grinning: